### PR TITLE
Update combiner-explainer.md

### PR DESCRIPTION
**Pull request by sreecha**

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @sreecha isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
The Linux Foundation CLA GitHub bot_

File: doc/combiner-explainer.md [hunk 1]
```diff
@@ -119,37 +119,10 @@ while (q.pop(&f)) {
...
 ```
 
-`offload` is more than `break`; it does `break` but also causes some
-other thread that is currently waiting on a poll to break out of its
-poll. This is done by setting up a per-polling-island work-queue
-(distributor) wakeup FD. The work-queue is the converse of the combiner; it
-tries to spray events onto as many threads as possible to get as much concurrency as possible.
+`offload` offloads to the work to an `executor` which is an internal pool of threads. 
+`offload_combiner_work_to_some_other_thread` makes one of the threads in the
+executor to pick up and continue the work on the combiner.
```
Comment (vjpai) [inline doc/combiner-explainer.md]: Can you give pseudocode for offload and executor just like the old version had it for offload and workqueue?

**Comment by nicolasnoble** [comment]
@vjpai want to take this one over and close it?

**Comment by vjpai** [comment]
Sure, I can take it over

**Comment by sreecha** [comment]
For some reason, I was unable to resurrect this branch (maybe because my email associated with the github accout changed since I created this PR)

I created a new PR https://github.com/grpc/grpc/pull/18970 that is a dup of this PR (and also addresses vijay's comments)

**Pull request closed without merging**